queries/sflog/highlights.scm

```diff
@@ -0,0 +1,44 @@
+; highlights.scm
+(number) @number
+
+(identifier) @variable
+
+(version) @string.special
+
+(namespace) @variable
+
+(limit) @property
+
+(limit
+  (identifier) @string)
+
+(limit
+  (number) @number)
+
+(duration
+  (number) @number) @string
+
+(location
+  (number) @number) @string
+
+(event_detail
+  (event_detail_value) @string)
+
+(log_level_settings) @comment
+
+(log_level_setting
+  (component) @type)
+
+(log_level_setting
+  (log_level) @constant)
+
+(log_entry
+  (timestamp
+    (time) @function) @number)
+
+(log_entry
+  (location) @number)
```

Comment:
This should be removed, since location will be either a `(number)` (already covered) or `"EXTERNAL"` (which should be its own thing)